.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# Notes (delete this section)
+
+* For new APIs, follow http://aka.ms/apireview. This requires filing an issue
+  that outlines the new API first. Before we accept PRs, it muse be reviewed by
+  us and marked as `api-approved`.
+
+* Do not create tracking PRs with pending work. Submit the PR when you're ready
+  to review the changes with us. If you want to share work-in-progress PRs,
```

Comment:
Maybe "review the changes with us and believe the changes are ready to be merged", or something like that.